packages/quantic/cypress/e2e/default-2/generatedAnswer/generated-answer-expectations.ts

```diff
@@ -371,27 +372,34 @@ function generatedAnswerExpectations(selector: GeneratedAnswerSelector) {
 
     logOpenGeneratedAnswerSource(
       streamId: string,
-      citation: {id: string; permanentid: string}
+      citation: {
```

Comment:
Eventually, these tests will have to be migrated to Playwright.

IMO this doesn't necessarily have to be done right away, but I'm making the comment anways ;)

Reply:
yeah it's just it's failing otherwise :)